### ISSUE TITLE
CAN examples which use floating point round to the nearest integer

### ISSUE THREAD

**Issue by trstamper** [ni/nixnet-python#259]
Reproduction steps:
1. Connect two CAN ports named CAN1 and CAN2
2. go to nixnet_examples and run can_signal_conversion.py or can_signal_single_point_io.py
3. Insert a non-integer number when prompted
4. Note that the console says "the same values should be received", but the output is rounded to the nearest integer.

- OS: Windows 7
- Python version: Python 3.6.5 32bit
- NI-XNET version: 17.5.1

### Installed packages and version
nixnet 0.2.1
pip 9.0.3
setuptools 39.0.1
six 1.11.0

**Comment by jashnani** [comment]
It's rounding values to nearest int because the datatype on the Signal object used by those examples (`CANEventSignal1`) is an a signed int. You can confirm this by opening the database in an editor and checking the datatype value.

We should consider either modifying the database or the example to better communicate the point. Slight preference for modifying the database so we can show that float values are also supported for Signals.

**Comment by d-bohls** [comment]
If we modify the NIXNET_example database that ships with the NI-XNET driver, then the example won't work with older XNET drivers installed. We should probably add another database in "nixnet-python\nixnet_examples\databases" that has the structure we want for our example.

**Comment by d-bohls** [comment]
>I want to avoid having a special version of the example database specifically for Python.

Agreed, I don't think python should have its own special version of NIXNET_example. My thinking was that if we do in fact need floats (which are not contained anywhere in the NIXNET_example database) then we could add a new database with a cluster containing float signals for use with that example. Doing this would not be unprecedented. We already have on github a database named "custom_database.dbc" which only the "programmatic_database_usage.py" python uses.

Two of the python examples ask for floats; the rest ask for ints. I don't like the idea of doing rounding from float to int unless it contributes directly to the example's intended purpose. If floats/rounding aren't needed, perhaps we should just stop asking for floats and modify the two previously mentioned examples to request ints instead.

**Comment by jashnani** [comment]
We further discussed this in the grooming meeting and decided to add a comment in code explaining why we're coercing to type int.
